hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -130,6 +130,14 @@
   </description>
 </property>
 
+<property>
+  <name>hadoop.security.resolver.impl</name>
+  <value></value>
```

Comment:
Should we specify the default just in case?

Reply:
I agree that your suggestion sounds like a good idea.  I followed the example of `dfs.namenode.edits.qjournals.resolver.impl` which doesn't provide the value.